variants/backend-base/config/app.yml

```diff
@@ -0,0 +1,24 @@
+# Be sure to restart your server when you modify this file.
+#
+# Use this file to load non-sensitive app config from ENV. Config values here
+# will be loaded into `Rails.application.config.app`.
+#
+# Sensitive config should be put in `config/secrets.yml` (which will load it
+# into `Rails.application.secrets`)
+
+default: &default
+  # The default `From:` address to use for email sent by this application
+  # obviously isn't a secret per se, but configuring it here is convenient
+  mail_from: "<%= ENV['MAIL_FROM'] %>"
+
+development:
+  <<: *default
+
+test:
+  <<: *default
+
+staging:
+  <<: *default
```

Comment:
🐘 ideally we should actually only add this if a `capistrano` variant is enabled, because Heroku doesn't do `staging` so it isn't reasonable to assume all apps will have it

Reply:
I think we should keep as is because:

1. Heroku is the odd one out in not supporting a staging env out of the box - every other way of deploying Rails uses one AFAIK (not just capistrano based deploys). 
1. Having this here doesn't harm a Heroku deploy.
2. We don't currently have a template config variable for "deploying to heroku". I don't think this use-case is enough to add that complexity.

Reply:
I'm not sure that can be the case because [`Rails/UnknownEnv`](https://docs.rubocop.org/rubocop-rails/cops_rails.html#railsunknownenv) does not include `staging` by default, but its a while since I looked at this so can't remember.

I'm fine with whatever you think is worth doing